crates/forge/src/run_tests/maat.rs

```diff
@@ -0,0 +1,11 @@
+use forge_runner::package_tests::with_config::TestCaseConfig;
+
+#[must_use]
+pub fn ignore_fork_tests() -> bool {
+    std::env::var("SNFORGE_IGNORE_FORK_TESTS").is_ok_and(|v| v == "1")
+}
+
+#[must_use]
+pub fn is_test_case_ignored(case_config: &TestCaseConfig, skip_fork_tests_from_env: bool) -> bool {
+    case_config.ignored || (skip_fork_tests_from_env && case_config.fork_config.is_some())
+}
```

Comment:
```suggestion
#[must_use]
fn ignore_fork_tests() -> bool {
    std::env::var("SNFORGE_IGNORE_FORK_TESTS").is_ok_and(|v| v == "1")
}

#[must_use]
pub fn env_ignore_fork_tests(case_config: &TestCaseConfig) -> bool {
    ignore_fork_tests() && case_config.fork_config.is_some()
}
```

Reply:
Actually, I've refactored it in a bit simpler way:

```rust
#[must_use]
pub fn env_ignore_fork_tests() -> bool {
    std::env::var("SNFORGE_IGNORE_FORK_TESTS").is_ok_and(|v| v == "1")
}